src/index.tsx

```diff
@@ -12,7 +12,7 @@ const style = {
     transition: "background 0.25s ease-in-out",
     backgroundPosition: "center",
```

Comment:
@Djspaceg since we would move to using background, we can remove the more specific background properties here.